build/Build.Tests.cs

```diff
@@ -194,10 +129,134 @@ bool DoesSidHaveRightsToDirectory(string directory, WellKnownSidType sid, params
             
                 foreach (var installer in installers)
                 {
-                    TestInstallerPermissions(installer);
+                    TestWindowsInstaller(installer);
                 }
             });
         });
+
+    string GetTestName(AbsolutePath installerPath) => Path.GetFileName(installerPath).Replace(".msi", "");
+        
+    void TestWindowsInstaller(AbsolutePath installerPath)
+    {
+        Log.Information($"\n--------------------------------------\nTesting Installer {GetTestName(installerPath)}\n--------------------------------------");        
```

Comment:
Having this loud and proud was helpful during testing, so I left it. But am happy to dumb it down.

Reply:
I like the loud and proud :)